src/sidebar/combat-tracker.js

```diff
@@ -465,6 +475,44 @@ export default class YearZeroCombatTracker extends CombatTracker {
     Hooks.call(`${MODULE_ID}.${data.event}`, data);
   }
 
+  /* ------------------------------------------ */
+
+  /**
+   * Sets the turn's properties for the template.
+   * @param {*} data
+   * @param {*} turn
+   * @returns {Promise.<any>}
+   */
+  static #setTurnProperties(data, turn) {
+    const { id } = turn;
+    const combatant = data.combat.combatants.get(id);
+    const flags = combatant.flags[MODULE_ID];
+    // FIXME: Figure out why these turn up as flags.
+    delete flags.fastAction;
+    delete flags.slowAction;
```

Comment:
Also if anyone has any idea why when using `toggleActiveEffect` on the token a flag is set on the combatant under the module ID I'd appreciate the insight.

Reply:
I am using the `toggleActiveEffect` in the Vaesen module, and this is not happening. So, maybe there is some overload on the yze-combat that is doing that? 🤔

Reply:
I think it is added by line 293

Reply:
It's not 293. I have narrowed the issue down to being the toggleActiveEffect call (although it could be happening as a side effect somewhere else). If I do not call toggleActiveEffect then it does not happen at all. 

Gonna try removing the active part.

Reply:
The `active` property is not the culprit either.

Reply:
@aMediocreDad any chance this is being done by the Forbidden Lands system?

I saw this in the code `src/system/combat/combat.js:153`

```ts
	async nextRound() {
		await Promise.all(
			this.turns.map((c) =>
				c.actor.update({
					"flags.forbidden-lands.fast": false,
					"flags.forbidden-lands.slow": false,
				}),
			),
		);
		return super.nextRound();
	}
```